content/documentation.rst

```diff
@@ -69,12 +69,18 @@ This section is also used for other frequent questions sent to Support / Sales
 that are not otherwise covered in the main documentation.
 
 User guides can also be written to the more general audience depending on the
-content.
+content. It is a good idea to specifically list out who the audience is.
 
 If the page has a specific audience in mind, state the audience in the
 introduction of the page.
 
 
+About Sphinx
+============
```

Comment:
Do we need this as a high level section?

Maybe is ok to have it as a paragraph